ziti/edge/api/client.go

```diff
@@ -307,5 +306,10 @@ func decodeSession(resp *http.Response) (*edge.Session, error) {
 		pfxlog.Logger().WithError(err).Error("failed to decode session response")
 		return nil, err
 	}
+
+	for _, edgeRouter := range session.EdgeRouters {
+		delete(edgeRouter.Urls, "wss")
```

Comment:
Please also add "ws". I am currently prototyping that transport on behalf of pure JS tls connections.

Reply:
done